lib/ipmi_backend_utils.pm

```diff
@@ -106,14 +106,17 @@ sub setup_console_in_grub {
             # autoballoning is disabled since sles15sp1 beta2. we use default dom0_ram which is '10% of total ram + 1G'
             # while for older release, bsc#1107572 "This dom0 memory amount works well with hosts having 4 to 8 Gigs of RAM"
             # considering of one SUT in OSD with 4G ram only, we set dom0_mem=2G
-            my $dom0_mem_options = "";
+            my $dom0_options = "";
             if (is_sle('<=12-SP4') || is_sle('=15')) {
-                $dom0_mem_options = "dom0_mem=2048M,max:2048M";
+              $dom0_options = "dom0_mem=2048M,max:2048M";
+            }
+            if (get_var("ENABLE_SRIOV_PCI_PASSSHTROUGH")) {
+              $dom0_options .= " iommu=on";
```

Comment:
1. Does kvm kernel needs iommu=on ?
2. How do you handle cpus from different vendors ? Is this necessary ?

Reply:
for *1: no, it is for XEN only. 
*2: Have not verify on Intel machine yet, this commit implement on AMD machine first. When I get some more time slot, I will support Intel machines.